subprojects/core/src/main/java/org/gradle/internal/service/scopes/WorkerSharedGlobalScopeServices.java

```diff
@@ -144,11 +135,6 @@ Deleter createDeleter(Clock clock, FileSystem fileSystem, OperatingSystem os) {
         return new DefaultDeleter(clock::getCurrentTime, fileSystem::isSymlink, os.isWindows());
     }
 
-    @Provides
-    PropertyFactory createPropertyFactory(PropertyHost propertyHost) {
```

Comment:
💭 Removed since [WorkerSharedGlobalScopeServices.java](https://github.com/gradle/gradle/pull/31165/files#diff-996d8b1edde7b8d6c513d92c730fe99ce1ee04a50d3039c211f79c1ceeb0cd3d) extends [BasicGlobalScopeServices.java](https://github.com/gradle/gradle/pull/31165/files#diff-e7898653dd1b9fc002098bf5c32aa9d9323a51409ad1aa7782249e8567c6721f)